modules/custom/az_core/az_enterprise_attributes_import/az_enterprise_attributes_import.install

```diff
@@ -22,3 +22,10 @@ function az_enterprise_attributes_import_install() {
   }
 
 }
+
+/**
+ * Enable migrate_queue_importer module for dependency.
+ */
+function az_enterprise_attributes_import_update_1020801() {
```

Comment:
We aren't planning to create any more 2.8.x patch releases so I think we'll want to change the update number here to be in the `10209xx` series.

```suggestion
function az_enterprise_attributes_import_update_1020901() {
```